### ISSUE TITLE
peer review_midterm report

### ISSUE THREAD

**Issue by LimingZHAO911** [kathy908000/Population-Segmentation-For-Health-System-Management#6]
Three good aspects:
1.  The logistic of the midterm report is well organized. I can easily understand why they want to work on this data set, what the preprocessing is, which features should be chosen, and how the model works.
2. All features were well encoded based on their types. 
3. The argument for over-fitting and under-fitting is reasonable

Three aspects that can be improved:
1: In the final report, all the features can be more clearly summarized in a table.
2. What percentage of the training set and test set are allocated?
3. If wanting to focus on the inliers, Huber and quantile regression may outperform than least square regression.
